src/FluentHub.App/Strings/README.md

```diff
@@ -0,0 +1,12 @@
+# 😸 FluentHub Translations
+
+#### These are the translation/i18n resources for FluentHub
```

Comment:
IMO we don't need this

Reply:
🤷 Just so that people don't accidentally update localisation directly, and do on Crowdin instead. I don't see why we should remove.

Reply:
Because why not!